feedback/static/html/studio_view.html

```diff
@@ -23,6 +23,22 @@
             <span class="tip setting-help">{% trans "The display name for this component." %}</span>
         </li>
 
+        <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+                <label class="label setting-label" aria-describedby="ff_p_div">{% trans "Message for posting feedback on third-party platform" %}</label>
+                <input class="input setting-input" name="external_feedback_msg" id="external_feedback_msg" value="{% trans external_feedback_msg %}" type="text"/>
```

Comment:
It's a bad usage of `trans` tag `{% trans external_feedback_msg %}`. I think we need to remove `trans` tag in this case.

Reply:
I don't like that also. But in this HTML template,  `trans` template tag was used for all form fields. I followed one code style.

Reply:
That's bad usage. Please, check this style guide - [doc](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#only-translate-literal-strings).
It's up to you.

Reply:
Thanks. You are right) I removed tag `trans` for this case.